polaris-ui/src/app/features/cases/api/reauthentication-filter.ts

```diff
@@ -21,14 +22,24 @@ const tryCleanRefreshIndicator = (window: Window) => {
   }
 };
 
-const tryHandleFirstAuthFail = (response: Response, window: Window) => {
+const tryHandleFirstAuthFail = (
+  response: Response,
+  window: Window,
+  correlationId?: string
+) => {
   if (isCmsAuthFail(response) && !isAuthPageLoad(window)) {
     const delimiter = window.location.href.includes("?") ? "&" : "?";
 
-    const nextUrl = `${REAUTH_REDIRECT_URL}${encodeURIComponent(
+    let nextUrl = `${REAUTH_REDIRECT_URL}${encodeURIComponent(
       window.location.href + delimiter + REAUTHENTICATION_INDICATOR_QUERY_PARAM
     )}`;
 
+    if (correlationId) {
+      nextUrl += encodeURIComponent(
+        `${delimiter + FAIL_CORRELATION_ID_QUERY_PARAM}=${correlationId}`
```

Comment:
You probably just want a `&` here instead of `delimiter`. `delimiter` can be `?` if there are no query params already in the URL